pages/community/community-contribute-to-precice.md

```diff
@@ -98,15 +98,30 @@ Your case may already fit into one of the existing tutorials. If not, feel free
   - Which versions of preCICE, adapters, and solvers have you tried?
   - If it is a solver we don't already support, how can we get it?
   - How should the results look like? A screenshot would be very helpful.
-- In the `README.md` file, document the scenario setup, the dependencies, how to run it, how to visualize the results, and an example picture or video of the results. Follow the general structure in the existing tutorials. Don't forget to adapt the `permalink:` field in the beginning of the file.
-- The run scripts (`run.sh`) should be very short. You can probably reuse some of the scripts we already provide.
+- In the `README.md` file, following the general structure of the existing tutorials, document:
+  - the scenario setup,
+  - the dependencies,
+  - how to run the tutorial,
+  - how to visualize the results, and
+  - an example picture or video of the results.
+  - Don't forget to adapt the `permalink:` field in the beginning of the file.
+- The run scripts (`run.sh`) should be very short.
+  - Try to reuse the scripts we already provide in `tools/` and in other tutorials.
+  - The run script itself must be executable for a default case without any arguments.
+  - Optional arguments can include `-parallel`, or anything that triggers a special case.
 - For the `clean.sh` script, you can use the functions provided in `tools/cleaning-tools.sh`
+- In case a tutorial-specific code example is needed for this tutorial, and this can be reused among participants, add that in `solver-<code>/`.
+  - You don't need to have a participant-specific configuration file in a participant case. A `run.sh` that provides the participant as parameter is enough. 
```

Comment:
I am not sure I get the last sentence.

Reply:
Clarified. The `oscillator-overlap` already does this:

```sh
python3 ../solver-python/oscillator.py Mass-Left
```

and the case directory only includes `clean.sh` and `run.sh`, without any other configuration files. This is totally fine and enough, since still the `run.sh` can be executed without additional parameters and show the expected behavior.